.github/workflows/phpunit.yml

```diff
@@ -83,18 +83,21 @@ jobs:
         options: --health-cmd=pg_isready --health-interval=10s --health-timeout=5s --health-retries=3
 
       mssql:
-        image: mcr.microsoft.com/mssql/server:2019-CU28-ubuntu-20.04
+        image: mcr.microsoft.com/mssql/server:2022-latest
```

Comment:
there is the 2025 image but i'm not sure if that's stable yet

Reply:
Same as the above comment.